include/config/config_game.h

```diff
@@ -4,5 +4,17 @@
 /*****************
  * GAME SETTINGS *
  *****************/
+ 
+/*****************
+*** Allocation ***
+/****************/
+/*******************************************
+*GI models or "Get Item" models are*********
+*the models that appear over Link's head****
+*when he picks up a item. the normal memory*
+*allocated for the is 0x3008. a safe number*
+*is 1024 * 32 which is way more than normal*
+*******************************************/
```

Comment:
```suggestion
/*************************************************************
 * GI ("Get Item") models, are those that appear             *
 * over Link's head when he picks up an item.                *
 * The normal memory allocated for the is 0x3008.            *
 * A safe number is 1024 * 32, which is way more than normal.*
 *************************************************************/
```
Looks better like that imo, also I think it'd be nice to precise the unit, I assume it's bytes but I wasn't sure so I didn't mention it in the suggestion

Reply:
looks better. i also used 1024 * 32 because an exact value may seem like a weird arbitrary value to newcomers.

Reply:
like 32768 as opposed to 1024 * 32

Reply:
i also think 32 kb might be overkill but its like the biggest value i tested and seems safe. bigger hack may need smaller sizes.

Reply:
I still think we should add the unit in the comment

Reply:
i could but zel said he may do this dynamically so ima wait for him

Reply:
Can you test #18 to see if it works for you with your larger GI models? @AriaHiro64